### PR TITLE
test: Move TestIDs from page objects in e2e/pages folder

### LINKED ISSUE #7721: Move TestIDs from page objects in e2e/pages folder
We've encountered cases in which the testIDs are located within the page objects rather than being housed in the designated testID folder. This can potentially perplex those who are new to the automation code. To address this issue, the proposed solution involves relocating the TestIDs from each page object to their corresponding testID file within the testID folder. This task is aimed at transferring the TestIDs from the page objects in the e2e/pages folder.

Here are the following e2e/pages you should work on:

AccountListView
ActivitiesView
AddCustomTokenView
ContractNickNameView
ImportAccountView
ImportTokensView
LoginView
RequestPaymentView
TransactionConfirmView
WalletView

### Technical Details

- The testID folder lives in wdio/testID. Work would be done later on to move this to the e2e folder. See ticket here: https://app.zenhub.com/workspaces/metamask-mobile-5f984938ddc0e4001d4b79cb/issues/gh/metamask/metamask-planning/1546
- Move the testIDs in each page object in the e2e/pages folder to the appropriate testID folder. 
- Remove duplicate testID's when necessary
- The centralized location of all the testIDs files would be e2e/selectors
- File could lived in subfolders following the same structure as e2e/pages
- Files would follow *.selectors.js file mask
- Selectors would be group in objects, each for a different strategy of the selectors needed for the page object. For example:
```
// Main TabBarSelectors object
const TabBarSelectors = {
  WALLET_BUTTON_ID: 'tab-bar-item-Wallet',
  WALLET_BUTTON_XPATH: '//[@a;dslfja;]',
  BROWSER_BUTTON_ID: 'tab-bar-item-Browser',
  BROWSER_BUTTON_CSS: 'div nth(5)',
  ACTION_BUTTON_ID: 'tab-bar-item-Actions',
  ACTION_BUTTON_CLASS: 'button',
  SETTING_BUTTON_ID: 'tab-bar-item-Setting',
  ACTIVITY_BUTTON_ID: 'tab-bar-item-Activity',
};

// Object for ID selectors. Shared between app code and test code
 const TabBarSelectorsIDs = {
  WALLET_BUTTON_ID: TabBarSelectors.WALLET_BUTTON_ID,
  BROWSER_BUTTON_ID: TabBarSelectors.BROWSER_BUTTON_ID,
  ACTION_BUTTON_ID: TabBarSelectors.ACTION_BUTTON_ID,
  SETTING_BUTTON_ID: TabBarSelectors.SETTING_BUTTON_ID,
  ACTIVITY_BUTTON_ID: TabBarSelectors.ACTIVITY_BUTTON_ID,
};

// Object for CSS selectors
 const TabBarSelectorsByCSS = {
  WALLET_BUTTON_CSS: TabBarSelectors.BROWSER_BUTTON_CSS,
};

// Object for XPATH selectors
 const TabBarSelectorsByXPATH = {
  WALLET_BUTTON_XPATH: TabBarSelectors.WALLET_BUTTON_XPATH,
};
 
// Object for Text Selectors
 TabBarTextSelectors
 
export { TabBarSelectorsByID, TabBarSelectorsByCSS, TabBarSelectorsByXPATH };
```

### Acceptance Criteria

- ALL E2E tests should pass. There should be no failing tests. 
- TestIDs must be completely removed from page objects within the e2e/pages folder. 
- TestIDs used in mostly on the e2e/pages from app/constants/test-ids.js to the selectors folder

### PR DESCRIPTION
**Pull request by SamuelSalas**
## **Description**
 Relocate the TestIDs from each page object to their corresponding testID file within the testID folder. This task is aimed at transferring the TestIDs from the page objects in the e2e/pages folder.

## **Related issues**

Fixes: #7721

## **Manual testing steps**

1. Go to this page...
2.
3.

## **Pre-merge author checklist**

- [ ] I’ve followed [MetaMask Coding Standards](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/CODING_GUIDELINES.md).
- [ ] I've clearly explained what problem this PR is solving and how it is solved.
- [ ] I've linked related issues
- [ ] I've included manual testing steps
- [ ] I've included screenshots/recordings if applicable
- [ ] I’ve included tests if applicable
- [ ] I’ve documented my code using [JSDoc](https://jsdoc.app/) format if applicable
- [ ] I’ve applied the right labels on the PR (see [labeling guidelines](https://github.com/MetaMask/metamask-mobile/blob/main/.github/guidelines/LABELING_GUIDELINES.md)). Not required for external contributors.
- [ ] I’ve properly set the pull request status:
  - [ ] In case it's not yet "ready for review", I've set it to "draft".
  - [ ] In case it's "ready for review", I've changed it from "draft" to "non-draft".

## **Pre-merge reviewer checklist**

- [ ] I've manually tested the PR (e.g. pull and build branch, run the app, test code being changed).
- [ ] I confirm that this PR addresses all acceptance criteria described in the ticket it closes and includes the necessary testing evidence such as recordings and or screenshots.

### CODE REVIEW

File: e2e/pages/ImportTokensView.js
Comment (chrisleewilcox) [inline e2e/pages/ImportTokensView.js:1]: @SamuelSalas update this file that's currently on main.

**Review by chrisleewilcox — CHANGES_REQUESTED** [review]
See comment.

File: e2e/pages/ImportTokensView.js
Comment (SamuelSalas) [reply e2e/pages/ImportTokensView.js:1]: Done

**Review by chrisleewilcox — DISMISSED** [review]
LGTM

File: .tool-versions
Comment (chrisleewilcox) [inline .tool-versions:1]: This change here is not currently in main.  Try to keep PRs limited to only your changes.  Please update this branch with current main and if there are changes from your branch that is not part of the PR and it's not in sync with main then revert those changes.

**Review by chrisleewilcox — DISMISSED** [review]
Please see comments.

File: .tool-versions
Comment (SamuelSalas) [reply .tool-versions:1]: Just remove it.

File: e2e/specs/browser/browser-tests.spec.js [hunk 1]
```diff
@@ -84,10 +83,7 @@ describe(Smoke('Browser Tests'), () => {
       await TestHelpers.tapAtPoint(BROWSER_SCREEN_ID, { x: 180, y: 275 });
       await Browser.waitForBrowserPageToLoad();
     }
-    await TestHelpers.checkIfElementWithTextIsVisible(
-      METAMASK_TEST_DAPP_SHORTEN_URL_TEXT,
-      0,
-    );
+    await Browser.isWebViewTestDapp();
```
Comment (cortisiko) [inline e2e/specs/browser/browser-tests.spec.js:86]: This is checking the URL bar for the short hand url for the test dapp right? We should rename it to reflect that. First place I assumed we are asserting that the test dapp loads correctly.

**Review by cortisiko — CHANGES_REQUESTED** [review]
I only left one small comment. Also, run the regression pipeline in addition to the smoke pipeline on this PR.

**Comment by SamuelSalas** [comment]
> I only left one small comment. Also, run the regression pipeline in addition to the smoke pipeline on this PR.

**Review by cortisiko — APPROVED** [review]
✅ ✅ ✅

**Pull request merged**